### DISCUSSION TITLE
Make it harder to accidentally abort a cherry-pick

### DISCUSSION THREAD

**Discussion by ccasin** [magit/magit#5051] [Ideas]
While in the middle of applying a commit (after `A A`), you can press `A` to get options for how to proceed:
```
Actions
  A Continue
  s Skip
  a Abort
```
Note that `A` and `a` are very similar keys and do very different things.  Further, there is no confirmation prompt when aborting.  This makes it pretty easy to apply a commit, spend a while fixing up conflicts, and then accidentally throw all that work into the void.  And I think the work will be totally unrecoverable (there was no commit with it so it won't show up in the reflog).

I'd like to suggest that the "Continue" and "Abort" keys should be different letters, or abort should require confirmation, or both.

**Comment by tarsius** [comment]
Agreed. Done in fb1a3c487eabe51f2bf5f80cb9a6d9e43172f8b0.

We already do that before aborting a rebase.

**Reply by ccasin** [reply]
Thanks for the quick implementation!  Appreciated!
